packages/connect/src/implementation.ts

```diff
@@ -56,16 +56,43 @@ interface MI<
   readonly idempotency?: MethodIdempotency;
 }
 
-// TODO document
+/**
+ * Context for an RPC on the server. Every RPC implementation can accept a
+ * HandlerContext as an argument, and
```

Comment:
Should there be additional text here after `, and...`?

Reply:
TY